docs/src/options.md

```diff
@@ -254,7 +254,7 @@ Options:
   - false (default)
 
 ```javascript
-session_serialization true
+pane_viewport_serialization true
```

Comment:
It's actually `serialize_pane_viewport` (should be updated in the header as well)

Reply:
Done. Thank you.